11-payment-encoding.md

```diff
@@ -115,7 +115,7 @@ A writer MUST include exactly one `p` field, and set `payment_hash` to
 the SHA-2 256-bit hash of the `payment_preimage` which will be given
 in return for payment.
 
-A writer MUST NOT include more than one `d`, `h`, or `x` fields, and
+A writer MUST NOT include more than one `d`, `h`, `n` or `x` fields, and
```

Comment:
This has been fixed in 80688d79008e9e999dff485f4dc03fcda3300291, the PR needs a rebase